Source/driver/sdc_util.F90

```diff
@@ -81,6 +83,13 @@ subroutine sdc_solve(dt_m, U_old, U_new, C, sdc_iteration)
 
        ! failing?
        if (ierr /= NEWTON_SUCCESS) then
+          print *, "Newton convergence failure"
```

Comment:
Doesn't look like U_orig is being filled here

Reply:
fixed